### ISSUE TITLE
 Create echomail area failed

### ISSUE THREAD

**Issue by askovpen** [ftnapps/FTNd#1]
after `ftnfido toss` in debug.log:
> ? 07-Dec-2014 01:51:03 ftnfido[12131] Create echomail area SOME.NAME failed
\+ 07-Dec-2014 01:51:03 ftnfido[12131] Tossing in bad board

why?

how to enable autocreate?

**Comment by jame** [comment]
Found some issues with the code which have resolved with updates now released with FTNd v0.19.  I have a test install of that version operational on a Debian system (1:120/419), where the only time "Tossing in bad board" happens (that I've seen so far) is when the echo isn't actually defined in the system.  Closing as resolved.

**Comment by askovpen** [comment]
```
  16-May-2018 17:02:08 ftnfido[27752] FTNFIDO v0.25.0                                                                                                                                                       
  16-May-2018 17:02:08 ftnfido[27752] Cmd: ftnfido to                                                                                                                                                       
+ 16-May-2018 17:02:08 ftnfido[27752] Pass: toss netmail (/opt/ftnd/var/inbound)                                                                                                                            
+ 16-May-2018 17:02:08 ftnfido[27752] Pass: toss ARCmail (/opt/ftnd/var/inbound)                                                                                                                            
+ 16-May-2018 17:02:08 ftnfido[27752] Execute: /u01/app/oracle/product/11.2.0/xe/bin/unzip -o -j -L fc39ff00.we0                                                                                            
+ 16-May-2018 17:02:09 ftnfido[27752] Packet   : 5afc39a5.pkt type 2+                                                                                                                                       
+ 16-May-2018 17:02:09 ftnfido[27752] From     : 2:5020/9696 to 2:5020/9696.128                                                                                                                             
+ 16-May-2018 17:02:09 ftnfido[27752] Dated    : 16-05-2018 17:01:09                                                                                                                                        
+ 16-May-2018 17:02:09 ftnfido[27752] Program  : HPT 1.9                                                                                                                                                    
? 16-May-2018 17:02:09 ftnfido[27752] Create echomail area RU.UNIX.BSD failed                                                                                                                               
+ 16-May-2018 17:02:09 ftnfido[27752] Tossing in bad board                                                                                                                                                  
? 16-May-2018 17:02:09 ftnfido[27752] Create echomail area RU.UNIX.BSD failed                                                                                                                               
+ 16-May-2018 17:02:09 ftnfido[27752] Tossing in bad board                                                                                                                                                  
+ 16-May-2018 17:02:09 ftnfido[27752] Messages : 2                                                                                                                                                          
+ 16-May-2018 17:02:09 ftnfido[27752] Flushing outbound queue                                                                                                                                               
+ 16-May-2018 17:02:09 ftnfido[27752] Echomail [   2] import [   0] out [   0] bad [   2] dupe [   0]                                                                                                       
m 16-May-2018 17:02:09 ftntask[25910] MIB set echo 4,2,0,2,0;                                                                                                                                               
m 16-May-2018 17:02:09 ftntask[25910] MIB echo: in=90 out=0 bad=67 dupe=11 in=90 out=0 bad=67 dupe=11                                                                                                       
  16-May-2018 17:02:09 ftnfido[27752] FTNFIDO finished in  1.00s                                                                                                                                            
- 16-May-2018 17:02:09 ftntask[25910] Unregistered client pgm "ftnfido", pid 27752, slot 1, mailers 0, TCP/IP 0                                                                                             
- 16-May-2018 17:04:12 ftntask[25910] Registered client pgm "ftnfido", pid 27824, slot 1, mailers 0, TCP/IP 0                                                                                               
```
